docs/photoniq/vwrs/vwrs-metrics.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_position: 100
+title: Get VWRs Metrics
+---
+
+Macrometa PhotonIQ Virtual Waiting Rooms (VWRs) has built-in metrics that allow you to retrieve waiting room telemetry per waiting room, per minute. Data is stored for the past 90 days.
```

Comment:
This got me thinking:  VWR vs VWRS.

What is the S?  "Get the VWRs metrics" ==> "Get the Virtual Waiting Room metrics"  What happened to the "s"

I think originally the S was Service ==> Virtual Waiting Room Service (is that right @rahul-naik1)

Should then this start with "Macrometa PhotonIQ Virtual Waiting Room Service..."
and then what about  "Rooms" 

@diana-macrometa you know, me overthinking probably, but thought I would just say it out loud.  You do with it what you think is best.

Reply:
The "s" just makes it plural.

Reply:
@nicolewilliams88 and @macrochrissie , any comment?